src/main/java/draughts/Game.java

```diff
@@ -9,13 +9,21 @@ public class Game {
 
     private Board board;
     private int drawCondition = 15;
-    private int sizeBoard;
 
-    public Game(int sizeBoard) {
-        this.board = new Board(sizeBoard);
-        this.sizeBoard = sizeBoard;
-    }
 
+    public Game() {
+        this.board = new Board(getBoardSizeFromUser());
+    }
+    private static int getBoardSizeFromUser() {
```

Comment:
we should use blank line between methods. Like it is suggested in [Oracle docs](https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html)